cupy/core/_routines_statistics.pyx

```diff
@@ -206,27 +206,27 @@ nanmax = create_reduction_func(
 
 cdef _argmin = create_reduction_func(
     'cupy_argmin',
-    ('?->q', 'B->q', 'h->q', 'H->q', 'i->q', 'I->q', 'l->q', 'L->q',
-     'q->q', 'Q->q',
-     ('e->q', (None, 'my_argmin_float(a, b)', None, None)),
-     ('f->q', (None, 'my_argmin_float(a, b)', None, None)),
-     ('d->q', (None, 'my_argmin_float(a, b)', None, None)),
-     ('F->q', (None, 'my_argmin_float(a, b)', None, None)),
-     ('D->q', (None, 'my_argmin_float(a, b)', None, None))),
+    tuple(['{}->{}'.format(d, r) for r in 'bhilq' for d in '?BhHiIlLqQ'])
```

Comment:
I am a bit concerned with the order `'bhilq'`. Which return type is chosen when `argmin(..., dtype=None, ...)` is called? I thought we'd need to put `'q'` as the leading character to match NumPy default, but I could be wrong. (I don't have access now to test this PR.)

Reply:
Thank you for pointing out! It seems you are correct. Some of the tests were failing.